client/state/ui/editor/video-editor/reducer.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	VIDEO_EDITOR_RESET_STATE,
+	VIDEO_EDITOR_SET_POSTER_URL,
+	VIDEO_EDITOR_SHOW_ERROR,
+	VIDEO_EDITOR_SHOW_UPLOAD_PROGRESS,
+} from 'state/action-types';
+
+/**
+ * Tracks poster updated state.
+ *
+ * @param {Boolean} state Whether or not the video poster was already updated
+ * @param {Object} action Action object
+ * @returns {Boolean} Whether or not the video poster is now updated
+ */
+export const isPosterUpdated = ( state = false, action ) => {
+	switch ( action.type ) {
+		case VIDEO_EDITOR_SET_POSTER_URL:
+		case VIDEO_EDITOR_SHOW_ERROR:
+			return true;
```

Comment:
is the poster really updated if we got an error? seems like a semantic mismatch between the data point and its use